TUM-Dev/NavigaTUM
# Added "Official Roomfinder" badge

Final change to get a TUM-corpoprate communication centre design licence.
This is one of the requirements to shut down the mytum roomfinder.

## Proposed Changes (include Screenshots if possible)

- Adds a "Official Roomfinder" to the bottom
- Removes the TUM logo from the header (this is a proposal by them)

## How to test this PR

1. See the footer in the staging
2. Resize the page to see a responsive footer

## How has this been tested?

- as above

## Checklist
